### DISCUSSION TITLE
What this even means?

### DISCUSSION THREAD

**Discussion by ghost** [PowerShell/PowerShell#18306] [General]
I was trying to update PowerShell before "oh-my-posh" and enhancing my command prompt/terminal since I just finished doing so on my laptop. This is occurring on my desktop. I am running Windows 11 btw. Any help, please?

![Screenshot 2022-10-15 015027](https://user-images.githubusercontent.com/75822760/195971286-74f33720-37d4-45c1-bfef-b19518ff0aa7.jpg)

**Comment by SeeminglyScience** [comment]
You may want to try asking on the oh-my-posh repo itself, but at a glance I'd guess you're doubling up on `Invoke-Expression` somewhere. That looks like you're trying to invoke the actual result prompt string like it's a script when you should be just returning it

**Reply by ghost** [reply]
Thank you for the reply! This makes a lot of sense! I will take care of this.
